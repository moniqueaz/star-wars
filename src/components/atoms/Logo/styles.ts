import styled from 'styled-components';

export const Image = styled.img`
  width: 15rem;
  height: auto;
  
  @media(min-width: 1024px){
    width: 25rem;
  }
`;
