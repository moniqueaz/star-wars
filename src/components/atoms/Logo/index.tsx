import * as S from './styles';

const Logo = () => (
  <S.Link href="/">
    <S.Image src="/img/logo.png" alt="logo star wars" />
  </S.Link>
);

export default Logo;
