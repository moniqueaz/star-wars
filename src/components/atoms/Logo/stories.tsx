import { Story, Meta } from '@storybook/react/types-6-0';
import Logo from '.';

export default {
  title: 'Atoms/Logo',
  component: Logo,
} as Meta;

export const Default: Story = () => {
  return <Logo />;
};
