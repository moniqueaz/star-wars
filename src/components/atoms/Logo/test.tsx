import { render, screen } from '@testing-library/react';
// import { renderWithTheme } from 'utils/tests/helpers';

import Logo from '.';

describe('<Logo />', () => {
  it('should render the heading', () => {
    const { container } = render(<Logo data-testid="Logo"/>);
    expect(screen.getByAltText('logo star wars')).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  } );
} );
