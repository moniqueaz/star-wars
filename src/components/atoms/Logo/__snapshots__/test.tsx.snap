// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render 1`] = `
.c0 {
  width: 15rem;
  height: auto;
}

@media (min-width:1024px) {
  .c0 {
    width: 25rem;
  }
}

<img
  alt="logo star wars"
  class="c0"
  src="/img/logo.png"
/>
`;
